solarwinds_apm/apm_config.py

```diff
@@ -594,7 +596,7 @@ def update_with_env_var(self) -> None:
         available_envvs = set(self.__config.keys())
         # TODO after alpha: is_lambda
         for key in available_envvs:
-            if key in {"inst_enabled", "transaction", "inst"}:
+            if key == "transaction":
```

Comment:
Is there a reason for having `transaction` in `self.__config.keys()`. I think removing it from there will nullify this branch.

Reply:
It's there as part of one way to store `transaction` with a default and `transaction.prepend_domain_name` as a nested object if set by customer, but I'm not convinced this is a good reason!

Right now an accepted config file would be:

```
{
    "transaction": {
        "prependDomain": false
    },
    "transactionSettings": []
}
```

But it might make more sense to customer if it were:
```
{
    "prependDomain": false,
    "transactionSettings": []
}
```

I'll change this in a next PR.